src/porepy/models/fluid_mass_balance.py

```diff
@@ -368,9 +369,19 @@ class BoundaryConditionsSinglePhaseFlow:
 
     """
 
-    mdg: pp.MixedDimensionalGrid
-    pressure: Callable[[list[pp.Grid]], pp.ad.Operator]
-    darcy_flux: Callable[[Sequence[pp.Grid]], pp.ad.Operator]
+    bc_data_darcy_flux_key: str = "darcy_flux"
+    """TODO"""
+    bc_data_pressure_key: str = "pressure"
```

Comment:
I think we should rather reuse the variable name.

Reply:
The variable name is not available at the initialization time of this class, because it's initialized in the init method of `SolutionStrategySinglePhaseFlow`. I can also add the init method for the boundary condition class, but it restricts the rule of thumb, that mixins must not have their own init methods. 

I agree with reusing it, so I move the definition of the name as the private global variable inside the same file and refer to it in both places. Please, let me know if you see a more elegant solution.

Reply:
No, I mean to remove the attribute `bc_data_pressure_key` and use `pressure_variable` instead. Optionally renaming to `pressure_(variable_)key`

Reply:
Ahhh, got it